metricbeat/module/kubernetes/controllermanager/_meta/fields.yml

```diff
@@ -0,0 +1,124 @@
+- name: controllermanager
+  type: group
+  description: >
+    Controller manager metrics
+  release: ga
+  fields:
+    - name: handler
+      type: keyword
+      description: >
+        Request handler
+    - name: code
+      type: keyword
+      description: >
+        HTTP code
+    - name: method
+      type: keyword
+      description: >
+        HTTP method
+    - name: host
+      type: keyword
+      description: >
+        Request host
+    - name: name
+      type: keyword
+      description: >
+        Name for the resource
+    - name: zone
+      type: keyword
+      description: >
+        Infrastructure zone
+    - name: process
+      type: group
+      fields:
+        - name: cpu.sec
+          type: scaled_float
+          description: CPU seconds
+        - name: memory.resident.bytes
+          type: long
+          description: Bytes in resident memory
+        - name: memory.virtual.bytes
+          type: long
+          description: Bytes in virtual memory
+        - name: fds.open.count
+          type: integer
+          description: Number of open file descriptors
+        - name: started.sec
+          type: scaled_float
+          description: Seconds since the process started
+    - name: http
+      type: group
+      fields:
+        - name: request.duration.us.percentile.*
+          type: scaled_float
+          description: Request duration microseconds percentiles
+        - name: request.duration.us.sum
+          type: scaled_float
+          description: Request duration microseconds cumulative sum
+        - name: request.duration.us.count
+          type: long
+          description: Request count for duration
+        - name: request.size.bytes.percentile.*
+          type: integer
```

Comment:
Any reason to use `integer` in these percentiles, but `long` in counts? I think we should use `long`.
We could also consider using `double` in all these numeric metrics as they come from prometheus where metrics are `float64`.

Reply:
did not know we should default to these.
I'll change those types